site/docs/.vuepress/configs/en.ts

```diff
@@ -376,6 +376,47 @@ export const localeEn: LocaleConfig<DefaultThemeLocaleData> = {
 export const docsearchEn: LocaleConfig<DocsearchLocaleData> = {
   "/": {
     placeholder: "Search",
-    translations: { button: { buttonText: "Search" } },
+    translations: {
+      button: {
+        buttonText: "Search",
+        buttonAriaLabel: "Search",
+      },
+      modal: {
+        searchBox: {
+          resetButtonTitle: "Clear the query",
+          resetButtonAriaLabel: "Clear the query",
+          cancelButtonText: "Cancel",
+          cancelButtonAriaLabel: "Cancel",
+        },
+        startScreen: {
+          recentSearchesTitle: "Recent",
+          noRecentSearchesText: "No recent searches",
+          saveRecentSearchButtonTitle: "Save this search",
+          removeRecentSearchButtonTitle: "Remove this search from history",
+          favoriteSearchesTitle: "Favorite",
+          removeFavoriteSearchButtonTitle: "Remove this search from favorites",
+        },
+        errorScreen: {
+          titleText: "Unable to fetch results",
+          helpText: "You might want to check your network connection.",
+        },
+        footer: {
+          selectText: "to select",
+          selectKeyAriaLabel: "Enter key",
+          navigateText: "to navigate",
+          navigateUpKeyAriaLabel: "Arrow up",
+          navigateDownKeyAriaLabel: "Arrow down",
+          closeText: "to close",
+          closeKeyAriaLabel: "Escape key",
+          searchByText: "Search by",
+        },
+        noResultsScreen: {
+          noResultsText: "No results for",
+          suggestedQueryText: "Try searching for",
+          reportMissingResultsText: "Believe this query should return results?",
+          reportMissingResultsLinkText: "Let us know.",
+        },
+      },
+    },
```

Comment:
Does this equal the default interface language for English?

Reply:
It does. I think it should be added as a reference for translators, even though the result is the same.

Reply:
I agree, I like these changes. I just wanted to be sure that we had the same understanding here. Thanks for clarifying it!